lib/rubocop/ast/node/if_node.rb

```diff
@@ -141,6 +141,35 @@ def node_parts
 
         [condition, true_branch, false_branch]
       end
+
+      # Returns an array of all the branches in the conditional statement.
+      #
+      # @return [Array<Node>] an array of branch nodes
+      def branches
```

Comment:
Implementation looks good. I think there might be some small enhancements that can be made to readability. Example:

```ruby
branches = [if_branch]

return branches unless else_branch

other_branches = if elsif_conditional?
                   else_branch.branches
                 else
                   else_branch
                 end

branches.concat(other_branches)
```

WDYT?

Reply:
@Drenmi 

Looks great! I've changed your example code a bit and replaced the code.

```ruby
branches = [if_branch]

return branches unless else_branch

other_branches = if elsif_conditional?
                   else_branch.branches
                 else
                   [else_branch] # changed here
                 end

branches.concat(other_branches)
```